galaxy/generate_options.py

```diff
@@ -0,0 +1,21 @@
+import sys
+import os
+
+# this add to path the home dir, so it can be called from anywhere
+sys.path.append(os.path.split(sys.path[0])[0])
```

Comment:
I don't like this, but it is necessary to be able to import contents of `MSMetaEnhancer` from this script. I have already run into this kind of issue in my previous project, but I don't know how to get rid of that. Any suggestions?

Reply:
Well I guess the "proper" approach would be to actually treat the script as having the MSME as a requirement, but in this case that is an overkill. The only people who will ever run this are probably us, so I think this is fine.

Reply:
Yeah, that is a good point. Alright, let's keep it that way.